census/templates/census/event_confirm_delete.html

```diff
@@ -12,7 +12,7 @@ <h3 class="usa-alert__heading">Warning you cannot get this event back</h3>
         <br/>
         <br/>
         <div class="grid-col-4 grid-offset-4">
-          <a class="usa-button usa-button--outline" href="/pending"> Go Back </a>
+          <button class="usa-button usa-button--outline" onclick="history.back()"> Go Back </button>
```

Comment:
This page is displayed when we try to delete an event from the **Review Events** or **Update Approved Events** page. So I thought it would make more sense that it should go back to the previous page as opposed to harcoding the /pending page.

Reply:
Note: I also noticed that even if I press 'Go Back' on the delete confirmation page, it still calls the delete handler and deletes the event anyway. I'll file a bug for this.